app/src/main/java/com/vero/tasky/agenda/data/mappers/AttendeeMapper.kt

```diff
@@ -0,0 +1,30 @@
+package com.vero.tasky.agenda.data.mappers
+
+import com.vero.tasky.agenda.data.local.entities.AttendeeEntity
+import com.vero.tasky.agenda.data.remote.network.dto.AttendeeDto
+import com.vero.tasky.agenda.domain.model.Attendee
+
+object AttendeeMapper {
```

Comment:
I'm personally a friend of just making the mappers extension functions and not putting them in an `object`, that feels a bit more natural. For example:
```
fun AttendeeDto.toAttendeeEntity(): AttendeeEntity() {
    // ...
}
```